app/src/main/res/navigation/user_mobile_navigation.xml

```diff
@@ -4,7 +4,21 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/mobile_navigation"
-    app:startDestination="@+id/navigation_home">
+    app:startDestination="@+id/navigation_all_events">
+
+    <fragment
+        android:id="@+id/navigation_all_events"
+        android:name="com.example.pygmyhippo.admin.AllEventsFragment"
```

Comment:
Why are these included in the user navigation and not a new admin nav?

Reply:
For testing purposes. I will make a admin nav soon.